core/modules/wiki.js

```diff
@@ -384,24 +384,22 @@ exports.forEachTiddler = function(/* [options,]callback */) {
 };
 
 /*
-Return an array of tiddler titles that are directly linked from the specified tiddler
+Performs a method on each treeNode in the tiddler title, then returns the results of all calls in a list with no duplicates.
 */
-exports.getTiddlerLinks = function(title) {
+exports.collectFromEachTiddlerNode = function(title, cacheName, treeNodeMethod) {
```

Comment:
While I like the refactoring, the `getTiddlerLinks` function will probably need to stay. What you can do is reroute it to call `collectFromEachTiddlerNode` under the hood.